build-config/make/images.make

```diff
@@ -196,11 +196,17 @@ endif
 sysroot-check: $(SYSROOT_CHECK_STAMP)
 $(SYSROOT_CHECK_STAMP): $(PACKAGES_INSTALL_STAMPS)
 	$(Q) for file in $(SYSROOT_LIBS) ; do \
-		[ -r "$(DEV_SYSROOT)/lib/$$file" ] || { \
-			echo "ERROR: Missing SYSROOT_LIB: $$file" ; \
-			exit 1; } ; \
-		find $(DEV_SYSROOT)/lib -name $$file | xargs -i \
-			cp -av {} $(SYSROOTDIR)/lib/ || exit 1 ; \
+		if [ "$(ARCH)" == "arm64" ] || [ "$(ARCH)" == "x86_64" ] ; then \
```

Comment:
Why is x86_64 included here?  x86_64 is currently building fine as it is with the new crosstool-ng.

It seems like this is only an arm64 issue.  I'm probably missing something.